helm_deploy/court-case-matcher/templates/_envs.tpl

```diff
@@ -4,6 +4,9 @@ Environment variables for web and worker containers
 */}}
 {{- define "deployment.envs" }}
 env:
+  - name: SPRING_PROFILES_ACTIVE
```

Comment:
This won't exclude other spring profiles will it  ? Shouldn't be possible I suppose.

Reply:
this isn't being set anywhere else so it shouldn't previously have had any profiles applied

Reply:
sorry, you're quite right! Hadn't realised SPRING_PROFILES_ACTIVE was already being set